specifyweb/frontend/js_src/lib/localization/query.ts

```diff
@@ -638,22 +638,26 @@ export const queryText = createDictionary({
     `,
   },
   future: {
-    'en-us': 'Future',
+    'en-us': 'in the future',
   },
   past: {
-    'en-us': 'Past',
+    'en-us': 'in the past',
   },
   day: {
-    'en-us': 'Day',
+    'en-us': 'Days',
   },
   week: {
-    'en-us': 'Week',
+    'en-us': 'Weeks',
   },
   month: {
-    'en-us': 'Month',
+    'en-us': 'Months',
   },
   year: {
-    'en-us': 'Year',
+    'en-us': 'Years',
+  },
+  relativeDate: {
+    'en-us':
+      '<input>{size:number}</input> <select>{type:string}</select> <secondSelect>{direction:string}</secondSelect>',
```

Comment:
@CarolineDenis can you fix this on this branch:
Mainly:
Add a comment for translators clarifying what this is used for
Replace the HTML-like tag names with clearer tag names (i.e, magnitude, size, direction)